python/xorbits/_mars/lib/nvutils.py

```diff
@@ -494,12 +494,23 @@ def get_device_status(dev_index: int) -> _nvml_device_status:
         if util_node.find("gpu_util").text == "N/A":
             gpu_util = 0
         else:
-            gpu_util = int(util_node.find("gpu_util"))
+            try:
+                gpu_util = int(util_node.find("gpu_util"))
+            except TypeError:
+                # Solve the problem of illegal conversion from string to int
+                gpu_util = int(util_node.find("gpu_util").text.split()[0])
```

Comment:
What if line 501 raised an error?

Reply:
I integrated them into the "try" branch.